src/t_hash.c

```diff
@@ -661,6 +661,11 @@ void hincrbyCommand(client *c) {
     }
     value += incr;
     new = sdsfromlonglong(value);
+
+    if (sdslen(new) > server.hash_max_listpack_value)
+        hashTypeConvert(o, OBJ_ENCODING_HT);
+    hashTypeTryConversion(o, c->argv, 2, 2);
```

Comment:
Is this redundant?

Reply:
hashTypeTryConversion is not NOP.It is used to check whether the length of the field exceeds the limit.It is used to check whether the length of the field exceeds the limit.Because hincrbyfloat can also create a new field-value.

Reply:
You are right, perhaps we need a `else`.

Reply:
Done.Thanks for your advice.